src/lib.rs

```diff
@@ -172,6 +172,57 @@ struct Wallet {
     wallet_mutex: Mutex<BdkWallet<AnyDatabase>>,
 }
 
+pub struct OutPoint {
+    txid: String,
+    vout: u32,
+}
+
+pub struct TxOut {
+    value: u64,
+    address: String,
+}
+
+pub enum KeychainKind {
+    External = 0,
+    Internal = 1,
+}
+
+impl From<bdk::KeychainKind> for KeychainKind {
+    fn from(x: bdk::KeychainKind) -> KeychainKind {
+        match x {
+            bdk::KeychainKind::External => KeychainKind::External,
+            bdk::KeychainKind::Internal => KeychainKind::Internal,
+        }
+    }
+}
+
+pub struct LocalUtxo {
```

Comment:
The BDK version of this struct has a boolean `is_spent`. I assume you didn't include it here because `LocalUtxo` will only be returned from the `list_unspent()` method, which means that boolean will only ever been set to `false`. In this case, however, I think the naming could lead to confusion if some developers are familiar with the Rust version of BDK.

I propose we either:
1. Rename the struct `UnspentLocalUtxo` to make explicit the fact that it is unspent, or
2. Add the `is_spent` boolean to it, which for now will only ever be false

I suggest we go with (2) because to me that feels more forward-compatible. if ever we implement other features that use the `LocalUtxo` struct we'll have it in its entirety already. I also feel like it might allow you to simplify some of the mapping between the `LocalUtxo` struct you have here and the one defined in BDK.

Reply:
I also feel that (2) is a better choice here so just added the field